viz-lib/src/visualizations/chart/Editor/ColumnMappingSelect.jsx

```diff
@@ -12,9 +12,18 @@ const MappingTypes = {
   zVal: { label: "Color Column" },
 };
 
-export default function ColumnMappingSelect({ value, availableColumns, type, onChange }) {
+export default function ColumnMappingSelect({ value, availableColumns, type, onChange, isAxesInverted }) {
   const options = sortBy(filter(uniq(flatten([availableColumns, value])), v => isString(v) && v !== ""));
-  const { label, multiple } = MappingTypes[type];
+
+  let { label, multiple } = MappingTypes[type]
+  // this inverts the ui, as the data will be inverted on render
+    if (isAxesInverted) {
+      if (type === "x") {
+        label = label.replace("X", "Y");
+      } else if (type === "y") {
+        label = label.replace("Y", "X");
+      }
+    }
```

Comment:
Something in the formatting looks off, worth running Prettier here.

Reply:
`npm run prettier` currently doesn't include the `viz-lib` 😅, I'll fix that.

---

@rafawendel I think for it to work in vscode you'll need to manually specify the settings path to `client/prettier.config.js` for the workspace.